### ISSUE TITLE
Reproducability

### ISSUE THREAD

**Issue by PatGlan** [andyrull/width-and-Depth-pruning-for-Vision-Transformer#5]
I just run your method with Deit-T for 100 epochs. Unfortunately I do not get the reported Top-1_accurracy that's been reported.

I run your script with following command (batchsize=256 to match the total batchsize of 1024): 
"python -m torch.distributed.launch --nproc_per_node=4 --master_port $used_port --use_env main_wdpruning.py  --data-set IMNET --data-path my/path --batch-size 256 --arch deit_tiny --classifiers 10 --R_threshold 0.7"

my Results:
Deit_tiny (R-thresh=0.7, classifiers=10 in training, but use 12 classifiers in evaluation): top1_acc=62.376%
Deit_tiny (R-thresh=0.7, classifiers=12 in training and evaluation): top1_acc=62.192%

In your paper you reported a top1-accuracy for this setting of 71.10% for Deit_tiny, when pruning 0.3% in width and use classifer 12. Why is there such an offset? Did you use other hyperparameters for Deit_tiny?

Best Patrick
